R/internal.R

```diff
@@ -2,3 +2,93 @@
     # works like message() but prepends a timestamp
     message(date(), ": ", ...)
 }
+
+.rgbaConversion <- function(color, alpha = 0.5){
+    rgbastr <- NULL
+    if (!is.character(color)) {
+        rgbastr <- "invalid value"
+    }
+    tryCatch({
+        rgbaVal <- paste(c(col2rgb(color), alpha), collapse = ",")
+        rgbastr <- paste0("rgba(", rgbaVal, ")")
+    }, error = function(e) {
+        warning(paste("Ïnvalid color specified",color))
+    })
+    if (is.null(rgbastr)) {
+        rgbastr <- "invalid value"
+    }
+    return(rgbastr)
```

Comment:
no need for explicit return call

Reply:
@agenius-mohammed-ali  - This one is not working if there is not explicit call. That is why I left it as is